app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/check_eori_details.scala.html

```diff
@@ -35,12 +35,12 @@
     formWithCSRF: uk.gov.hmrc.govukfrontend.views.html.helpers.formWithCSRF
 )
 
-@(userDetails : SignedInUserDetails, form:Form[CheckEoriDetailsAnswer], postAction : Call, link : Call)(implicit request: RequestWithSessionData[_], messages: Messages, viewConfig: ViewConfig)
+@(userDetails : SignedInUserDetails, form:Form[CheckEoriDetailsAnswer], postAction : Call)(implicit request: RequestWithSessionData[_], messages: Messages, viewConfig: ViewConfig)
```

Comment:
Is it okay to remove a back link? 
The reason is that this is the first page which user sees once `start` call been executed, so there is no landing page to go back.